Detectors/data/ldmx-det-v14-8gev-no-cals/detector.gdml

```diff
@@ -0,0 +1,229 @@
+    <variable name="recoil_pos_z" value="9*mm + recoil_envelope_dz/2"/>
+
+    <!-- Position of the ECal and HCal parent volumes -->
+    <constant name="em_calorimeter_pos_z" value="ecal_front_z" />
+    <constant name="hadron_calorimeter_pos_z" value="ecal_front_z + hcal_dz/2." />
+
+    <!--
+      Trigger scintillator parent volume position relative to the target
+      ???? mm. The end of the parent volume relative to the target
+      is at ?????. This gives a ???? mm clearance between the upstream.
+      There are now two TS modules upstream of the tagger tracker.
+    -->
+    <variable name="trig_scint_pos_z" 
+      value="tagger_pos_z-trig_scint_area_envelope_z/2 -tagger_envelope_dz/2- clearance - 200" />
+    
+    <!-- 
+         Define all subdetector positions relative to the target.  
+         Note: The position of the target is defined at the center i.e. 
+         x = y = z = 0 mm.     
+    -->
+    <position name="trig_scint_pos" x="0.0" y="0.0" z="trig_scint_pos_z"/>
+    <position name="tagger_pos" x="0.0" y="0.0" z="tagger_pos_z"/>
+    <position name="recoil_pos" x="0.0" y="0.0" z="recoil_pos_z"/>
+    <position name="em_calorimeter_pos"
+		  x="0.0" 
+		  y="support_box_shift"
+		  z="em_calorimeter_pos_z + ecal_envelope_z/2"/>
+    <!-- the ecal support box is asymmetric in the vertical direction causing an additional
+    shift relative to the origin at the target -->
+    <position name="hadron_calorimeter_pos"
+		  x="0.0" 
+		  y="support_box_shift"
+		  z="hadron_calorimeter_pos_z"/>
+    <position name="magnet_pos" x="0.0" y="0.0" z="magnet_pos_z" />
+	
+  </define>
+    
+  <materials>
+    &materials;
+  </materials>
+
+  <solids>
+    <box lunit="mm" name="world_box" x="world_dim" y="world_dim" z="world_dim"/>
+  </solids>
+    
+  <structure>
+    <volume name="World">
+      <materialref ref="Vacuum"/>
+      <solidref ref="world_box"/>
+      <physvol copynumber="1">
+        <file name="tagger.gdml"/>
+        <positionref ref="tagger_pos"/>
+        <rotationref ref="identity"/>
+      </physvol> 
+      <physvol copynumber="2">
+        <file name="target.gdml"  />
+        <positionref ref="center" />
+        <rotationref ref="identity"   /> 
+      </physvol> 
+      <physvol copynumber="3">
+        <file name="trig_scint.gdml"/>
+        <positionref ref="trig_scint_pos"/>
+        <rotationref ref="identity"/>
+      </physvol>
+      <physvol copynumber="5">
+        <file name="recoil.gdml"/>
+        <positionref ref="recoil_pos"/>
+        <rotationref ref="identity"/>
+      </physvol>
+      <!-- 
+      <physvol copynumber="6"> 
+        <file name="ecal.gdml"/> 
+        <positionref ref="em_calorimeter_pos"/> 
+        <rotationref ref="identity"/> 
+	    </physvol>
+      <physvol copynumber="7"> 
+        <file name="hcal.gdml"/> 
+        <positionref ref="hadron_calorimeter_pos"/> 
+        <rotationref ref="identity"/> 
+      </physvol> -->
```

Comment:
```suggestion
```
I looked at the diff wrt to the base and seeing only the commenting makes it less obs, while the full removal makes it clear